nicos_ess/loki/gui/loki_scriptbuilder.py

```diff
@@ -310,15 +310,30 @@ def _handle_delete_cells(self):
     def _handle_copy_cells(self):
         indices = [(index.row(), index.column())
                    for index in self.tableView.selectedIndexes()]
-        if len(set(
-            [len(list(group))
-             for _, group in groupby(indices, lambda x: x[0])])) != 1:
+
+        if not self._is_contiguous_selection(indices):
             # Can only select one continuous region to copy
+            self.showError("Multi-selection is not allowed during copying data")
```

Comment:
```suggestion
            self.showError("Selected data must be contiguous")
```
The message must be in language the user understands